templates/nginx.conf

```diff
@@ -40,7 +58,7 @@ server {
 
     location /media/ {
         alias {{ pretalx_media_dir }};
-        add_header Content-Disposition 'attachment; filename="$request_basename"';
+        add_header Content-Disposition 'attachment';
```

Comment:
Could you explain the reason for this change, please?

Reply:
Nginx fails to load configs with this line in it, saying that `$request_basename` isn't valid. This breaks the Ansible run, too, since it can't start Nginx.